__tests__/cities.test.js

```diff
@@ -0,0 +1,86 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('backend-express-template routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('#GET /cities should return a list of cities', async () => {
+    const resp = await request(app).get('/cities');
+    expect(resp.status).toBe(200);
+    expect(resp.body).toEqual([
```

Comment:
Great job! Considering you may work with data-sets too large to have in your test file, or project - its good to consider that you can also test this by grabbing just the first item in the resp.body array, and making sure it has the properties and values you're expecting. Rather than taking the time to match each item in the array.